### DISCUSSION TITLE
Font color change with quotes

### DISCUSSION THREAD

**Discussion by jlgrunr262** [vkbo/novelWriter#1806] [Help] [answered]
I just started using novelWriter.  I am on version 2.3.1 and running it on Ubuntu 23.10.  I started inputting text and found that when I use single or double quotes the font color between the quotes changes to blue.  I looked through the documentation, which is very good, and could not find an answer.  I don't know if this is a feature or something with my implementation. Thanks

**Comment by vkbo** [comment, marked as answer]
It's dialogue highlighting, which is quite useful for fiction writing. You can disable it in Preferences under "Text Highlighting" if you don't want it.

**Reply by vkbo** [reply]
I'll add something about it to the documentation as well.

**Reply by jlgrunr262** [reply]
I actually like it, so I'll keep it especially since I know it is a feature, not my linux.  Thanks!

**Reply by vkbo** [reply]
There will be a number of improvements to this feature in 2.5. Right now it just highlights text in quotes, but it will be more specialised to recognise dialogue for various languages and styles.

**Comment by jlgrunr262** [comment]
Thanks.  I am re-entering text I've already written. There is a lot to this
app. The good thing is I can use just a little or a lot. So far I am very
impressed.  I am happy to provide any feedback.

**Comment by jlgrunr262** [comment]
i am enjoying getting to know novelWriter. My approach is to just start
writing and then start learning more when I hit a roadblock and can't do
something I want to do. For me it is about getting the words on the page
with minimum distraction.

I have always written my first draft on paper or on a manual typewriter and
then re-keyed it into a word processor. There is something about the
tactile feel of pen on paper or the resistance of a typewriter key that
forces you to think of every word and take a little more time. I like an
app that respects that mode of thought capture. Maybe novelWriter will
convert me to direct input. The benefit of handwriting or typing is there
are no delete keys and fonts to worry about. No distractions.  Fortunately
you have an option to remove distractions. I learned to type decades ago in
a high school class on a manual typewriter. It is in my blood.

A benefit I see, and it is better than Scribner, is being able to add
markdown tags and notations. I often wondered why markdown wasn't being
used in apps for writing. You cracked the code and I think it gives your
app a lot of headroom.

Thanks for listening. I will give my self 6 weeks of solid use before I
make a final decision, but I am very impressed so far.

**Reply by vkbo** [reply]
I'm looking into adding an editor mode where you cannot edit the text you type, only move forward. I know a lot of writers prefer this, and it is something that has been requested. I think I would implement it so that you can only edit the end of the text. This would allow you to backspace if you pressed the wrong key.

**Comment by jlgrunr262** [comment]
That would be an interesting feature as long as you can escape it easily.
Would you tie it to a hot key combination? Like a Cntrl-e or something?  I
know that I like being able to do simple functions with keystrokes versus
using a mouse.  The beauty of a feature like the one you mention is it will
force writers to take a few milliseconds more with each key they press. I
think that is why I like a manual typewriter I have to do a little slower
than on a computer keyboard.

On a related note, what language hare you coded novelWriter in?  I am
amazed that you've built this very deep app as a side effort. What made you
start down this path?  If you don't have time to answer, don't worry.  I
won't take it personally.

Jerry
